cluster/kubemark/pre-existing/config-default.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
It would be good if you can add a short readme inside `test/kubemark/pre-existing`.